docs/api/archeryutils.baseclasses.rst

```diff
@@ -1,10 +1,16 @@
 Base Classes
 ============
+.. autoclass:: archeryutils.targets.Quantity
 
 .. autoclass:: archeryutils.Target
    :members:
    :undoc-members:
    :show-inheritance:
+..
+   Getting warnings for duplicate links being generated for class properties
+   that are also referenced in the class docstring.
+   Eg distance, diameter.
+   This is beyond my limited sphinx experience here.
```

Comment:
No problem, just pushed https://github.com/jatkinson1000/archeryutils/pull/59/commits/8c49fd752c4033320728ec9d9bef4b02bce88ab1 to this branch which should solve. It was because they were still defined as attributes but have been turned into properties of the class :)

Also made the internal checking data private so as not to clutter the docs with it.

Reply:
Thanks, that makes sense, I wansn't sure how the numpy docstring standards interacted with sphinx in the case of properties. 
Although the supported_x variables do kind of make sense as private, I don't know if this is best here. I think it also makes sense to expose those either for introspection or in the docs so that users can easily check what units/unit aliases are supported, without having to go read the units (convert) module to figure out whats going on there.

Reply:
That's fair, feel free to revert those back and see how it looks. :)

Reply:
Looking at leaving as private but documenting distance/diameter units. I left out supported_systems as thats already explicitly covered in the __init__ docstring and has its own type associated so shows up again in the scoring_system property.

This way I can either include them in the autoclass directive or seperately describe them below, have tried implementing the second in the next set of commits.